tests/test_encoding.py

```diff
@@ -26,22 +26,21 @@ def test_encoding2(v):
     assert not v.found_dead_code_or_error
 
 
-def test_non_utf8_encoding(v, tmpdir):
+def test_non_utf8_encoding(v, tmp_path):
```

Comment:
I think the tmpdir name is needed as-is for pytest to work (it automatically creates a temporary directory).

Reply:
`tmp_path` does the same thing. It's like the pathlib compatible replacement for tmpdir. But got to figure out if it works different on windows.

Reply:
Ah, thanks!